opencsp/common/lib/deflectometry/ImageProjection.py

```diff
@@ -281,6 +300,20 @@ def show_calibration_image(self):
         # Display with black border
         self.display_image_in_active_area(array)
 
+    def zeros(self) -> np.ndarray:
+        """
+        Creates a black image to fill the entire projected area.
+
+        Returns:
+        --------
+        image : np.ndarray
+            A 2D image with shape (self.size_y, self.size_x, 3), filled with zeros
+        """
+        # Create black image
+        black_image = np.zeros((self.size_y, self.size_x, 3), dtype=self.display_data['projector_data_type'])
+
+        return black_image
+

```

Comment:
Love this method, thanks for adding it. 
Just a couple of picky things: 
1. In the doc string, could we change "entire projected area" to "entire active area," or something similar to fit with the existing nomenclature better (see `display_image_in_active_area()`?
2. How do you feel about changing the method name to something more descriptive like `array_black_active_area()` or something like that?

Reply:
1. Yep, can do!
2. I chose "zeros" to match "np.zeros", but I can definitely change this to something else if you think that would be more appropriate. Maybe "get_black_array_active_area"?

Reply:
Also it's totally my fault that the size x/y attributes are confusing; `size_win_x` is the size of the entire window, and `size_x` is just the size of the active area. If we wanted to clean that up, that might be appropriate, but don't feel obligated to do that in this PR.

Reply:
Yeah, matching `np.zeros` is totally legit. I like having "active_area" somewhere in the name though (I can be convinced either way though). I do like your suggestion above, `array_black_active_area()`!

Reply:
Changed to `get_array_black_active_area` to match the "action" style of method naming conventions.

I agree that renaming `size_x`/`size_y` should happen at some point, just not in this PR.